internal/dump/dump.go

```diff
@@ -0,0 +1,40 @@
+package dump
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v8"
+	"go.riyazali.net/sqlite"
+)
+
+type dump struct {
+	rdb *redis.Client
+}
+
+func (f *dump) Args() int           { return -1 }
+func (f *dump) Deterministic() bool { return false }
+func (f *dump) Apply(ctx *sqlite.Context, values ...sqlite.Value) {
+	var (
+		key    string
+		result *redis.StringCmd
+		err    error
+	)
+	if len(values) == 1 {
+		key = values[0].Text()
+
+	} else {
+		err = fmt.Errorf("must input single arg to dump")
+		ctx.ResultError(err)
+		return
+	}
+	result = f.rdb.Dump(context.TODO(), key)
+
+	ctx.ResultText(fmt.Sprint(result.String(), result.FullName(), result.Name()))
```

Comment:
With the above change, this should just be:

```suggestion
	ctx.ResultText(result)
```